<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainScreen</class>
 <widget class="QDialog" name="MainScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>513</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main Screen</string>
  </property>
  <widget class="QPushButton" name="inventory">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Inventory</string>
   </property>
  </widget>
  <widget class="QPushButton" name="software">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Check Software</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>210</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loan">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>50</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Loan Agreement</string>
   </property>
  </widget>
  <widget class="QPushButton" name="work_request">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Work Request</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
